ocaml/xapi/xapi_vgpu.ml

```diff
@@ -17,12 +17,15 @@ open D
 (* Mutex to prevent duplicate VGPUs being created by accident *)
 let m = Mutex.create ()
 
-(* 1. Since multiple vgpus support, only device number = 0 or specify [11,31] since demu restriction.
-   2. device = 0 will requires xapi to give a valid number
-    - First sort the device number and the trying to find the next valid from range *)
+(* 1. Only device numbers in the range [0,20] are allowed due to space
+      limitations on the guest PCI bus.
+   2. The guest PCI bus slot will be equal to the device number plus 11.
+   3. device = 0 is recognised as a special value to automatically pick the next
+      available device number from the allowed range.
+*)
 
-let min_valid_vgpu_device = 11
-let max_valid_vgpu_device = 31
+let min_valid_vgpu_device = 0
+let max_valid_vgpu_device = 20
```

Comment:
In the future we could have more devices here for UEFI, if we start assignments from 8, but that requires the PCI slot assignment rework in XAPI.

Reply:
@edwintorok  the real pci address used in VM plugs 11, namely [11,31].